Sources/RInAppMessaging/TooltipDispatcher.swift

```diff
@@ -83,6 +87,11 @@ internal class TooltipDispatcher: TooltipDispatcherType, ViewListenerObserver {
             return
         }
 
+        let permissionResponse = permissionService.checkPermission(forCampaign: tooltip.data)
+        if permissionResponse.performPing {
```

Comment:
Please add `permissionResponse.display` property check and stop the display if the value is `false` (Just like in CampaignDispatcher)

Reply:
Added [here](https://github.com/rakutentech/ios-inappmessaging/pull/252/commits/bdc6005b7c443c62dbecbc34820596b726c718b0).